capa/v30.1.2/README.md

```diff
@@ -0,0 +1,30 @@
+# :zap: Giant Swarm Release v30.1.2 for CAPA :zap:
+
+This release reconfigures the Cilium HelmRelease to avoid disruptions during cluster upgrades.
+
+## Changes compared to v30.1.0
+
+### Apps
+
+- cluster-aws from v3.2.1 to v3.2.2
+- cilium from v0.31.1 to v0.31.3
+
+### cluster-aws [v3.2.1...v3.2.2](https://github.com/giantswarm/cluster-aws/compare/v3.2.1...v3.2.2)
+
+#### Changed
+
+- Chart: Update cluster to v2.2.1.
+
+### cluster [v2.2.0...v2.2.1](https://github.com/giantswarm/cluster/compare/v2.2.0...v2.2.1)
+
+#### Changed
+
+- Make HelmRelease options configurable per app.
+- Set Cilium HelmRelease timeout to 1hs and disable remediation.
+
```

Comment:
Shoud be under ### Componentsm, here's an example: https://github.com/giantswarm/releases/blob/d08a8a49fc99afe99fd85fcbf5a46d9ab3f80759/capa/v30.1.0/README.md

Reply:
And also `### cluster` hasn't been added, IMHO it would make sense to add the changes under `### cluster-aws`